src/components/ebay-chip/test/test.server.js

```diff
@@ -0,0 +1,16 @@
+import { use } from "chai";
+import { composeStories } from "@storybook/marko/dist/testing";
+import { snapshotHTML } from "../../../common/test-utils/snapshots";
+import * as stories from "../chip.stories"; // import all stories from the stories file
+const { Default, WithDelete } = composeStories(stories);
+const htmlSnap = snapshotHTML(__dirname);
+
+use(require("chai-dom"));
+
+it("renders default chip component", async () => {
```

Comment:
Can we rename these tests please?

1. The `default` chip is also static, so something like, `renders default static chip component`
2. The second one doesn't refer to chip; I think it should. Maybe something like, `renders interactive chip with close button when close label is provided`

Reply:
Also on this same line, we should group up all these tests in a `describe` and say: `ebay-chip` or something like that.
https://github.com/eBay/ebayui-core/blob/master/src/components/ebay-icon/test/test.server.js#L11
(I know that its not everywhere but I think we should start to add it in all places)

Reply:
Sounds good, I updated test names.